lib/geodb.js

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var geodb = require('fxa-geodb')()
+var ACCURACY_MAX_KM = 200
+var ACCURACY_MIN_KM = 25
+
+/**
+* Thin wrapper around geodb, to help log the accuracy
+* and catch errors. On success, returns an object with
+* `location` data. On failure, returns an empty object
+**/
+module.exports = function (log) {
+  return function (ip) {
+    return geodb(ip)
+      .then(function (location) {
+        var logEventPrefix = 'fxa.location.accuracy.'
+        var logEvent = 'no_accuracy_data'
+        var accuracy = location.accuracy
+
+        if (accuracy > ACCURACY_MAX_KM) {
+          logEvent = 'unknown'
+        } else if (accuracy > ACCURACY_MIN_KM && accuracy <= ACCURACY_MAX_KM) {
+          logEvent = 'uncertain'
+        } else if (accuracy <= ACCURACY_MIN_KM) {
+          logEvent = 'confident'
+        }
+
+        log.info({op: 'getGeoData', 'accuracy': accuracy})
+        log.info({op: 'getGeoData', 'accuracy_confidence': logEventPrefix + logEvent})
```

Comment:
I think we can also combine this into one log statement, something along the lines of

`log.info({op: 'getGeoData', 'geodb.accuracy': accuracy, 'geodb.accuracy_confidence': logEventPrefix + logEvent})`

@vladikoff that what you had in mind for name above?